### DISCUSSION TITLE
Noise echant

### DISCUSSION THREAD

**Discussion by rdhp330** [speechbrain/speechbrain#1629] [Q&A]
Is it possible to train this mtl-mimic-voicebank model to enhance noise rather than voice?
If it is possible how should the dataset be composed?

Another thing I didn't understand is, when we do new training is it transfer learning or new training?

Thanks!
Emanuele.

**Comment by mravanelli** [comment]
Hi Emanuele,
sorry for the late reply.
@pplantinga can be more precise than me here. If you change the loss, you can probably enhance the noise over rather than the speech. Not sure you have to change dataset, but you might need to set the noise as target instead of the speech part. The mtl-minic-voicebank needs multiple steps. At every stage, if you do not replace the model with your own one, we use a pre-trained model downloaded from the web.

**Comment by rdhp330** [comment]
> Hi Emanuele, sorry for the late reply. @pplantinga can be more precise than me here. If you change the loss, you can probably enhance the noise over rather than the speech. Not sure you have to change dataset, but you might need to set the noise as target instead of the speech part. The mtl-minic-voicebank needs multiple steps. At every stage, if you do not replace the model with your own one, we use a pre-trained model downloaded from the web.

Grazie Mirco. Why I should change the loss? It isn't enough to change the noise as target instead of the speech?
Thank you another time for the help.

**Comment by mravanelli** [comment]
Right, I think you can just swap the targets. This will work in a standard
speech enhancement recipe, but the mtl-mimic one is a bit trickier because
there is a speech recognizer as well in the loop. I'm not sure about the
role that the speech recognizer will play if the goal is to extract noise
rather than speech.

**Comment by pplantinga** [comment]
While in theory you can just swap noise targets for the old speech targets, in practice I wouldn't expect this to work very well. The noise data is much more diverse than the speech data, and could be hard to reproduce. I don't know what you're trying to achieve nor what data you'd like to use, but I wonder if it may be worth looking at this more from a separation standpoint than enhancement.

**Comment by rdhp330** [comment]
@mravanelli @pplantinga Thanks for the reply.
@pplantinga It is a good point! Maybe with the separation model I can achieve a good result.
I already use the model but I don't know how actually works; there is a documentation where I can understand how to organize the dataset for this train?

Thanks at all and sorry for the bad english !
